src/test/test_arch.cc

```diff
@@ -33,7 +34,7 @@ TEST(Arch, all)
 #if (__arm__ || __aarch64__ || __x86_64__) && __linux__
   char flags[FLAGS_SIZE];
   FILE *f = popen("grep '^\\(flags\\|Features\\)[	 ]*:' "
-                  "/proc/cpuinfo | head -1", "r");
+                  PROCPREFIX "/proc/cpuinfo | head -1", "r");
```

Comment:
@tchaikov 
Yes. My challenge is that the ARM code did work because ARM has enough FreeBSD support to compile. So I was going to figure out later on if this was really a Linux only case.
And then I would not have forgotten the PROCPREFIX.
But if you prefer me to take 'm out, I have no problem with that.